engineering-plans/0001-sql-query-combination.org

```diff
@@ -0,0 +1,397 @@
+# PLAN-0001: SQL Query Combination
+
+<dl>
+  <dt>Author</dt>
+  <dd>David Lutterkort</dd>
+
+  <dt>Implements</dt>
+  <dd>No RFC - no user visible changes</dd>
+
+  <dt>Engineering Plan pull request</dt>
+  <dd><a href="https://github.com/graphprotocol/rfcs/pull/2">https://github.com/graphprotocol/rfcs/pull/2</a></dd>
+
+  <dt>Date of submission</dt>
+  <dd>2019-11-27</dd>
+
+  <dt>Date of approval</dt>
+  <dd>2019-12-10</dd>
+
+  <dt>Approved by</dt>
+  <dd>Jannis, Leo</dd>
+</dl>
+
+This is not really a plan as it was written and discussed before we adopted
```

Comment:
Sadly, in contrast to GitHub, mdBook doesn't support Org Mode, so this will have to be converted to Markdown (let's drop the `<a>` anchors from the output though that Org Mode exports often include.